sample/set_cluster_location.py

```diff
@@ -0,0 +1,5 @@
+import rubrik_cdm
+
+RubrikConn = rubrik_cdm.Connect()
```

Comment:
`RubrikConn = rubrik_cdm.Connect()` should be `rubrik = rubrik_cdm.Connect()` to match the style of the other docs